### ISSUE TITLE
Tutorial is hard to follow

### ISSUE THREAD

**Issue by songproducer** [claudebarde/taquito-pinata-tezos-nft#5]
Hi Claude!

Assuming this is the tutorial:
https://wiki.tezos.com/build/create-a-tezos-token/nft

It looks like the tutorial is written for production but there's no instructions on setting up Heroku or any path/filenames for the code snippets.

Could you please add some steps to set up Heroku with the url please? Or how to get up and running on localhost?

I tried running npm install on macbook in frontend directory but got:

`npm ERR! path /Users/leotreasure/taquito-pinata-tezos-nft/frontend/node_modules/node-sass`

**Comment by claudebarde** [comment]
Hi!
I understand your frustration, however, this kind of tutorial is designed to only show you how to write an app to mint NFTs and assumes a certain knowledge of NPM and the JavaScript ecosystem. Writing a whole part about how to set up the app in a specific environment would take a lot more time and wouldn't be fair to other developers who need a setup in another environment (like on Vercel or Netlify instead of Heroku). Writing different tutorials for different environments is unfortunately not feasible.
You can find a lot of tutorials online to show you how to set up an NPM app on Heroku, if you need one-to-one assistance, I would be happy to assist you directly on Twitter or Telegram (contact me at @claudebarde)
